server/config/config.go

```diff
@@ -257,7 +257,9 @@ func AppConfig(denom string) (string, interface{}) {
 		customAppConfig.Config.MinGasPrices = "0" + denom
 	}
 
-	customAppTemplate := config.DefaultConfigTemplate + DefaultConfigTemplate
+	customAppTemplate := config.DefaultConfigTemplate +
+		DefaultEVMConfigTemplate //+
+		// memiavlcfg.DefaultConfigTemplate
```

Comment:
Is it correct we have this commented out? @MalteHerrmann

Reply:
hm this is rather something @GAtom22 can speak to with regards to MemIAVL - I would've expected us to still support this unless there was a change related to SDK v0.50 which makes this template unnecessary?

Reply:
for now it is commented out. When started the sdk v0.50 changes, memIAVL did not support this sdk version. Now it does, but will add the changes later